### ISSUE TITLE
AttributeError: module 'datetime' has no attribute 'UTC'

### ISSUE THREAD

**Issue by koxu1996** [casper-network/cctl#19]
While running CCTL built as docker:

```
2024-07-12T10:13:30.049491 [INFO] [1] CCTL :: network setup begins ... please wait
2024-07-12T10:13:30.050303 [INFO] [1] CCTL :: ... tearing down existing
2024-07-12T10:13:30.052865 [INFO] [1] CCTL :: ... setting assets directory
2024-07-12T10:13:30.145099 [INFO] [1] CCTL :: ... setting binaries
2024-07-12T10:13:30.485352 [INFO] [1] CCTL :: ... setting wasm payloads
2024-07-12T10:13:30.496459 [INFO] [1] CCTL :: ... setting cryptographic keys
2024-07-12T10:13:30.537823 [INFO] [1] CCTL :: ... setting supervisor config
2024-07-12T10:13:30.822927 [INFO] [1] CCTL :: ... setting genesis chainspec.toml
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'datetime' has no attribute 'UTC'
...
```

Chainspec setup fails because of it.

**Comment by koxu1996** [comment]
This error would not come up, if base images would be pin-pointed to specific versions.

**Comment by koxu1996** [comment]
### Problem analysis

`datetime.utcnow()` got deprecated in Python 3.12, and CCTL was updated in https://github.com/casper-network/cctl/commit/afa270f4f7c719c1ff512d27d3bfe4582da9ea63#diff-5c9309f1ddd513eeb901d8f457cdac17d69a89026109f62281452b48dd5f57aa. The problem is that change introduced usage of alias `datetime.UTC`, that is available since Python 3.11.

However, Dockerfile is based on `debian:buster` which currently ships Python 3.8, hence the error.

### Solution

There is no need to use such alias, we can replace it with `datetime.timezone.utc` that will work with all above mentioned Python versions.

Additionally container runtime could be updated too.
